Pidaq/Lib/dts_manager/dts_manager.py

```diff
@@ -0,0 +1,207 @@
+                self.bus.messages['0xa6'].data[0:2], byteorder='little', signed=True) / 10
+            self.phase_b_current = int.from_bytes(
+                self.bus.messages['0xa6'].data[2:4], byteorder='little', signed=True) / 10
+            self.phase_c_current = int.from_bytes(
+                self.bus.messages['0xa6'].data[4:6], byteorder='little', signed=True) / 10
+            self.dc_bus_current = int.from_bytes(
+                self.bus.messages['0xa6'].data[6:], byteorder='little', signed=True) / 10
+        if self.bus.messages['0xa8'].data:
+            self.id_feedback = int.from_bytes(
+                self.bus.messages['0xa8'].data[4:6], byteorder='little', signed=True) / 10
+            self.iq_feedback = int.from_bytes(
+                self.bus.messages['0xa8'].data[6:], byteorder='little', signed=True) / 10
+
+    def convert_angles(self) -> None:
+        if self.bus.messages['0xa5'].data:
+            self.motor_angle = int.from_bytes(
+                self.bus.messages['0xa5'].data[0:2], byteorder='little', signed=True) / 10
+            self.delta_filter_resolved = int.from_bytes(
+                self.bus.messages['0xa5'].data[6:], byteorder='little', signed=True) / 10
+
+    def convert_booleans(self) -> None:
+        BIT_0 = 1<<0
+        BIT_1 = 1<<1
+        BIT_2 = 1<<2
+        BIT_3 = 1<<3
+        BIT_4 = 1<<4
+        BIT_5 = 1<<5
+        BIT_6 = 1<<6
+        BIT_7 = 1<<7
+        if self.bus.messages['0xa4']:
+            digital_input_status = int.from_bytes(
+                self.bus.messages['0xa4'].data[0:], byteorder='little', signed=False)
+            self.digital_input_1 = bool(digital_input_status & BIT_0)
+            self.digital_input_2 = bool(digital_input_status & BIT_1)
+            self.digital_input_3 = bool(digital_input_status & BIT_2)
+            self.digital_input_4 = bool(digital_input_status & BIT_3)
+            self.digital_input_5 = bool(digital_input_status & BIT_4)
+            self.digital_input_6 = bool(digital_input_status & BIT_5)
+            self.digital_input_7 = bool(digital_input_status & BIT_6)
+            self.digital_input_8 = bool(digital_input_status & BIT_7)
+
+    def convert_torques(self) -> None:
+        if self.bus.messages['0xa2'].data:
+            self.torque_shudder = int.from_bytes(
+                self.bus.messages['0xa2'].data[6:], byteorder='little', signed=True) / 10
+        if self.bus.messages['0xac'].data:
+            self.commanded_torque = int.from_bytes(
+                self.bus.messages['0xac'].data[0:2], byteorder='little', signed=True) / 10
+            self.torque_feedback = int.from_bytes(
+                self.bus.messages['0xac'].data[2:4], byteorder='little', signed=True) / 10
+
+
+if __name__ == "__main__":
+    import can
+    bus = can_manager.CanManager('vcan0')
+    control = DTSControl(bus)
+    telemetry = DTSTelemetry(bus)
+    telemetry.bus.read_message_config('dts', 'message_config.json')
```

Comment:
Can this call be just bus.read_message_config('"dts", "message_config.json")

Reply:
indeed